### PR TITLE
added link validation in request

### LINKED ISSUE #10: Validate reference link in request (must start with https:// )

### PR DESCRIPTION
**Pull request by ameer-khan-ashraf**
close #10 
Added a function which checks if the link submitted has 'https://' at the 0th index on submission before it continues with the request. If the check fails the request returns ans an error message is displayed under the link input box.

### CODE REVIEW

File: src/pages/request/index.js [hunk 1]
```diff
@@ -58,9 +59,24 @@ function Request() {
     return urls;
   };
 
+  const validateLink = (link) =>{
+    if(link.indexOf("https://")==0){
+      return true;
+    }
+	  return false;
```
Comment (elliotBraem) [inline src/pages/request/index.js:66]: Run through a linter quick please, npm run lint

File: src/pages/request/index.js [hunk 2]
```diff
@@ -107,6 +123,11 @@ function Request() {
               referenceLink={referenceLink}
               setReferenceLink={setReferenceLink}
             />
+            {urlError && 
+              <div class="p-4 mt-2 text-sm text-red-700 bg-red-100 rounded-lg dark:bg-red-200 dark:text-red-800" role="alert">
+                <span class="font-medium"></span> Links must be secured and should start with https://
```
Comment (elliotBraem) [inline src/pages/request/index.js:128]: Should this text be within the span tags?

**Review by elliotBraem — CHANGES_REQUESTED** [review]
Good change, just noticed two items if you could please check first.
Also, seems your fork is behind master, mind pulling latest changes and then commit?

File: src/pages/request/index.js (hunk 1 above)
Comment (ameer-khan-ashraf) [reply src/pages/request/index.js:66]: Ran through the linter and had a few errors in the request page. Rectified them.

File: src/pages/request/index.js (hunk 2 above)
Comment (ameer-khan-ashraf) [reply src/pages/request/index.js:128]: I removed the span tag. Text looks better when not inside it.

**Review by elliotBraem — APPROVED** [review]

**Comment by elliotBraem** [comment]
Awesome! Thank you very much, merged and closing.

**Pull request merged**